cmds/ps/ps.go

```diff
@@ -49,6 +50,11 @@ func init() {
 	flag.BoolVar(&flags.all, "e", false, "Select all processes.  Identical to -A.")
 	flag.BoolVar(&flags.x, "x", false, "BSD-Like style, with STAT Column and long CommandLine")
 	flag.BoolVar(&flags.nSidTty, "a", false, "Print all process except whose are session leaders or unlinked with terminal")
+
+	if len(os.Args) > 1 {
+		flags.all = true
+		flags.aux = isPermutation(os.Args[1], "aux")
```

Comment:
a, u, and x are actually separate arguments to ps, aren't they? you can have "ps ax" "ps au" "ps aux" etc.